wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -173,6 +173,10 @@ public void tapDisplay (int x, int y) {
         getUiDevice().getInstance().click(x, y);
     }
 
+    public void pressBack() {
+        getUiDevice().getInstance().pressBack();
```

Comment:
Future: I think these methods should really be using the class name for static calls instead of the instance.